.github/workflows/ko-resolve.yaml

```diff
@@ -0,0 +1,86 @@
+name: ko resolve
+
+on:
+  push:
+    branches: [ publish-release ]
+
+jobs:
+
+  ko-resolve:
+    name: ko resolve
+    runs-on: ubuntu-latest
+    env:
+      GOPATH: ${{ github.workspace }}
+      GO111MODULE: off
+      KO_DOCKER_REPO: ko.local
+
+    steps:
+    - name: Set up Go 1.13.x
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13.x
+
+    - name: Add GOPATH/bin to PATH
+      shell: bash
+      run: |
+        echo "::set-env name=GOPATH::$(go env GOPATH)"
+        echo "::add-path::$(go env GOPATH)/bin"
+
+    - name: Install Dependencies
+      run: |
+        GO111MODULE=on go get github.com/google/ko/cmd/ko
```

Comment:
was this my issue?!

Reply:
I doubt it?  I added it for symmetry with `kn`, it was working for me before this.